pwiz_tools/Skyline/SettingsUI/Irt/CalibrateIrtDlg.cs

```diff
@@ -523,22 +545,34 @@ private void ShowGraph(string title, double[] xValues, double[] yValues, Diction
         public class RegressionOption
         {
             public string Name { get; }
-            public List<Tuple<DbIrtPeptide, PeptideDocNode>> MatchedPeptides { get; }
+            public RegressionLine RegressionLine { get; }
+            public List<Tuple<DbIrtPeptide, PeptideDocNode>> MatchedRegressionPeptides { get; }
+            public List<StandardPeptide> StandardPeptides { get; }
+            public bool AllowEditEquation { get; }

```

Comment:
It might be simpler to just have this property return "RegressionLine == null".
That is, you are allowed to edit the equation if the RegressionLine is null.

Reply:
Originally I had it like that, then made it a separate property. But you're right, I don't think there will be any case where we have a regression line and want it to be editable.